### PR TITLE
Update README.md; Added http://metricsparrow.com/toolkit/email-permut…

### PR DESCRIPTION
**Pull request by idrismusa4**
Added Metric Sparrow Email Permutator to email Section. Which Solvevs #160

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,8 +186,10 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Tomba.io](https://tomba.io/) - With 430+ million email addresses indexed, effective search filters, and deliverability checks, Tomba's email finder is its most powerful tool
 - [Snov Email Finder](https://snov.io/email-finder) Find any email. Anywhere
 - [EmailSearch.io](https://emailsearch.io/) - Find any emails and phones from a domain, Linkedin, name, and company
+- [MetricSparrow Email Permutator+](http://metricsparrow.com/toolkit/email-permutator/#) - This tool is intended to help you find the email addresses of people you need to contact by generating potential email addresses by permutating different combinations.
```
Comment (edoardottt) [inline README.md:189]: Please change to:
```
- [Email Permutator+](http://metricsparrow.com/toolkit/email-permutator/#) - Find potential email addresses permutating different combinations
```

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (idrismusa4) [reply README.md]: Okay done ✅

File: README.md [hunk 2]
```diff
@@ -187,10 +187,12 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Tomba.io](https://tomba.io/) - With 430+ million email addresses indexed, effective search filters, and deliverability checks, Tomba's email finder is its most powerful tool
 - [Snov Email Finder](https://snov.io/email-finder) Find any email. Anywhere
 - [EmailSearch.io](https://emailsearch.io/) - Find any emails and phones from a domain, Linkedin, name, and company
+- [Email Permutator+](http://metricsparrow.com/toolkit/email-permutator/#) - This tool is intended to help you find the email addresses of people you need to contact by generating potential email addresses by permutating different combinations.
```
Comment (edoardottt) [inline README.md:190]: it's not done...
```
- [Email Permutator+](http://metricsparrow.com/toolkit/email-permutator/#) - Find potential email addresses permutating different combinations
```

File: README.md [hunk 3]
```diff
@@ -187,10 +187,12 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
...
 - [Emailrep.io](https://emailrep.io/) - Simple Email Reputation
 - [Mailboxvalidator](https://www.mailboxvalidator.com/) - Secure and reliable email validation service to check for invalid email addresses
 - [ContactOut](https://contactout.com/) - Most accurate email finder for personal and work email outreach
 
+
```
Comment (edoardottt) [inline README.md:195]: unnecessary newline

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (idrismusa4) [reply README.md:190]: Should be good to go now

File: README.md [hunk 4]
```diff
@@ -187,9 +187,11 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Tomba.io](https://tomba.io/) - With 430+ million email addresses indexed, effective search filters, and deliverability checks, Tomba's email finder is its most powerful tool
 - [Snov Email Finder](https://snov.io/email-finder) Find any email. Anywhere
 - [EmailSearch.io](https://emailsearch.io/) - Find any emails and phones from a domain, Linkedin, name, and company
+- [Email Permutator+](http://metricsparrow.com/toolkit/email-permutator/#) - This tool is intended to help you find the email addresses of people you need to contact by generating potential email addresses by permutating different combinations.
```
Comment (edoardottt) [inline README.md:190]: This is duplicated here, remove it

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (idrismusa4) [reply README.md]: Okay you can check now

**Review by edoardottt — APPROVED** [review]
lgtm

**Comment by idrismusa4** [comment]
> lgtm

Okay then thanks for your revisions

**Pull request merged**